rtgui/editorpanel.cc

```diff
@@ -355,56 +354,51 @@ EditorPanel::EditorPanel (FilePanel* filePanel)
     vboxright->pack_start (*tpc->toolPanelNotebook);
 
     // Save buttons
-    Gtk::HBox* iops = Gtk::manage (new Gtk::HBox ());
-    iops->set_spacing(2);
-
-    Gtk::HBox *exportButtonsHBox = Gtk::manage (new Gtk::HBox ());
+    Gtk::Grid *iops = new Gtk::Grid ();
+    iops->set_orientation(Gtk::ORIENTATION_HORIZONTAL);
+    iops->set_row_spacing(2);
+    iops->set_column_spacing(2);
 
     Gtk::Image *saveButtonImage =  Gtk::manage (new RTImage ("gtk-save-large.png"));
     saveimgas = Gtk::manage (new Gtk::Button ());
     saveimgas->add(*saveButtonImage);
     saveimgas->set_tooltip_markup(M("MAIN_BUTTON_SAVE_TOOLTIP"));
+    setExpandAlignProperties(saveimgas, false, false, Gtk::ALIGN_CENTER, Gtk::ALIGN_CENTER);
 
     Gtk::Image *queueButtonImage = Gtk::manage (new RTImage ("processing.png"));
     queueimg = Gtk::manage (new Gtk::Button ());
     queueimg->add(*queueButtonImage);
     queueimg->set_tooltip_markup(M("MAIN_BUTTON_PUTTOQUEUE_TOOLTIP"));
+    setExpandAlignProperties(queueimg, false, false, Gtk::ALIGN_CENTER, Gtk::ALIGN_CENTER);
 
     Gtk::Image *sendToEditorButtonImage = Gtk::manage (new RTImage ("image-editor.png"));
     sendtogimp = Gtk::manage (new Gtk::Button ());
     sendtogimp->add(*sendToEditorButtonImage);
     sendtogimp->set_tooltip_markup(M("MAIN_BUTTON_SENDTOEDITOR_TOOLTIP"));
-
-    exportButtonsHBox->pack_start (*saveimgas, Gtk::PACK_SHRINK);
-
-    if(!simpleEditor) {
-        exportButtonsHBox->pack_start (*queueimg, Gtk::PACK_SHRINK);
-    }
-
-    exportButtonsHBox->pack_start (*sendtogimp, Gtk::PACK_SHRINK);
-
-    iops->pack_start (*exportButtonsHBox, Gtk::PACK_SHRINK);
-    iops->pack_start (*Gtk::manage(new Gtk::VSeparator()), Gtk::PACK_SHRINK, 1);
+    setExpandAlignProperties(sendtogimp, false, false, Gtk::ALIGN_CENTER, Gtk::ALIGN_CENTER);
 
     // Status box
-    statusBox = Gtk::manage (new Gtk::HBox ());
     cssProvider = Gtk::CssProvider::create();
-    progressLabel = Gtk::manage (new Gtk::ProgressBar());
+    progressLabel = Gtk::manage (new MyProgressBar(300));
     progressLabel->set_show_text(true);
-    setExpandAlignProperties(progressLabel, false, false, Gtk::ALIGN_CENTER, Gtk::ALIGN_CENTER);
+    setExpandAlignProperties(progressLabel, true, false, Gtk::ALIGN_START, Gtk::ALIGN_CENTER);
     progressLabel->set_fraction(0.0);
 
     // TODO is this needed? What for?
-    if (cssProvider) {
+    /*if (cssProvider) {
```

Comment:
If the code is commented out, why not fully delete it? We have version control so nothing is really lost and commented-out code only clutters things up and breaks when its surroundings are changed.